charts/actions-runner-controller/templates/githubwebhook.serviceMonitor.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   name: {{ include "actions-runner-controller-github-webhook-server.serviceMonitorName" . }}
+  namespace: {{ .Values.metrics.serviceMonitorNamespace }}
```

Comment:
@evan-hines-firebolt Hey!! Sorry for the radio silence. I was about to finish all the works towards the next version of ARC and finally had some time to review this once again.

In the middle of doing so, I realized that this may not work as flexibly as I first thought.

We use `namespace: {{ .Release.Namespace }}` for the `controller-manager` and `github-webhook-server` deployments and pods like declared in https://github.com/actions-runner-controller/actions-runner-controller/blob/4446ba57e1a21077136eb371795b7fe0acbc72c7/charts/actions-runner-controller/templates/githubwebhook.deployment.yaml#L6.

This `{{ .Values.metrics.serviceMonitorNamespace }}` doesn't prevent us from setting it to another namespace that's totally different from `{{ .Values.metrics.serviceMonitorNamespace }}`. Is it supposed to work? Perhaps not. In L22-24 we have this:

```
  selector:
    matchLabels:
      {{- include "actions-runner-controller-github-webhook-server.selectorLabels" . | nindent 6 }}
```

This basically says that this service monitor finds the target pods by labels within the same namespace as the service monitor. So, if `{{ .Values.metrics.serviceMonitorNamespace }}` wasn't equal to `{{ .Release.Namespace }}`, it will never able to monitor anything, right? 🤔 

Note that I'm not an expert in the prometheus-operator so your opinion and confirmation is very helpful!
If I'm not missing anything, probably we should update it to:

```suggestion
  namespace: {{ .Release.Namespace }}
```

and remove `metrics.serviceMonitorNamespace` as now unnecessary? WDYT?

Reply:
@evan-hines-firebolt Let's go ahead with my sugegstion! I'd appreciate it if you could submit a follow-up PR or a comment if this turns wrong.